lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -254,6 +254,27 @@ static bool printPackedTypeImpl(Type type, raw_ostream &os, Operation *op,
         op->emitError("Unexpected unpacked array in packed type ") << arrayType;
         return true;
       })
+      .Case<TypeAliasType>([&](TypeAliasType typeRef) {
+        SymbolRefAttr ref = typeRef.getRef();
+        auto moduleOp = op->getParentOfType<ModuleOp>();
+
+        auto typeScope =
```

Comment:
This seems like a common operation one would want. It'd be nice if it were a helper function in `TypeAliasType` which takes an op.